app/pdfs/print_record_pdf.rb

```diff
@@ -5,15 +5,15 @@ class PrintRecordPdf < Prawn::Document
   def initialize(record)
     super()
     @record = record
```

Comment:
I guess I don't really know what you mean. I see what you mean by removing that line, but then how will we access the record in methods other than initialize()?